src/Symfony/Component/Lock/Lock.php

```diff
@@ -70,6 +71,9 @@ public function acquire($blocking = false)
     {
         try {
             if ($blocking) {
+                if (!($this->store instanceof StoreInterface) && !($this->store instanceof BlockingStoreInterface && $this->store->supportsWaitAndSave())) {
```

Comment:
`!($this->store instanceof StoreInterface)` -> `!$this->store instanceof StoreInterface`